sys/shell/commands/sc_netif.c

```diff
@@ -73,30 +73,43 @@ static bool _is_iface(kernel_pid_t dev)
     return false;
 }
 
-#ifdef MODULE_NETSTATS_L2
-static int _netif_stats(kernel_pid_t dev, bool reset)
+#if defined(MODULE_NETSTATS_L2) || defined(MODULE_NETSTATS_IPV6)
```

Comment:
Just an idea: you could create a pseudo-module `MODULE_NETSTATS_PRINT` and let `MODULE_NETSTATS_L2`, `MODULE_NETSTATS_IPV6`, and hypothetically also future ones `MODULE_NETSTATS_UDP`, .. `MODULE_NETSTATS_X` depend on that. Then you can only check whether `MODULE_NETSTATS_PRINT` is defined instead of having a long and unreadable guard condition here.